tests/resources/trigger-data.json

```diff
@@ -0,0 +1,4090 @@
+[
+  {
```

Comment:
Does it make a difference that these are not camelCase like the original batchPlatesData.json data?